src/dialog.js

```diff
@@ -114,6 +114,19 @@ genotet.dialog.createNetwork_ = function() {
       var viewName = modal.find('#view-name');
       viewName.val(genotet.viewManager.nextSuffixName(viewName.val()));
       modal.find('.selectpicker').selectpicker();
+      var params = {
+        type: 'list-network'
+      };
+      $.get(genotet.data.serverURL, params, function(data) {
+          data.forEach(function(network) {
+            modal.find('.selectpicker')
+              .append('<option>' + network.networkName + '</option>');
```

Comment:
concating strings with html tags is not preferred. Please use
```js
$('<option></option>')
  .text(network.networkName)
  .appendTo(modal.find('.selectpicker'));
```

It would also be better to create a var for 
```js
var selectpicker = modal.find('.selectpicker');
```